components/raftstore/src/store/worker/read.rs

```diff
@@ -203,6 +203,11 @@ where
 {
     read_id: Option<ThreadReadId>,
     snap_cache: &'a mut SnapCache<E>,
+
+    // Used for case where read_id is not set, in which cases we don't use snapshot in case not
+    // releasing cache properly
```

Comment:
```suggestion
    // Used when read_id is not set, duplicated definition to avoid cache invalidation in case stale read
    // and local read are mixed in one batch.
```

Releasing the cache properly doesn't rely on duplicated definition.